snowflake/1.1/modules/ROOT/pages/snowflake-connector-studio.adoc

```diff
@@ -0,0 +1,142 @@
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Snowflake Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In *Mule Palette*, click *(X) Search in Exchange*.
+. In *Add Dependencies to Project*, type `snowflake` in the search field.
+. Click *Snowflake Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with Snowflake Connector:
+
+* Snowflake > On Table Row +
+Initiates a flow by selecting from a table at a regular interval and generating one message per obtained row
+* HTTP Listener +
+Initiates a flow each time it receives a request on the configured host and port
+* Scheduler +
+Initiates a flow when a time-based condition is met
+
+For example, to configure an *On Table Row* source, follow these steps:
+
+. In *Mule Palette*, select *Snowflake > On Table Row*.
+. Drag *On Table Row* to the Studio canvas.
+. On the *On Table Row* configuration screen, optionally change the value of *Display Name*.
+. Click *+* next to *Connector configuration* to configure a global element that can be used by all instances of the *On Table Row* source in the app.
+. On the *Snowflake Config* window, for *Required Libraries*, add the required Snowflake JDBC driver by selecting *Configure > Add recommended libraries* or *Configure > Add Maven dependency* and adding the artifact ID `snowflake-jdbc`.
+. On the *Connection* section, specify the database connection information for the connector.
+. On the *Advanced* tab, optionally specify connection pooling and reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specified Snowflake database.
+. Click *OK* to close the window.
+. On the *On Table Row* configuration screen, in *Table*, select the name of the table to select from.
+. On the *On Table Row* configuration screen, in *Watermark Column*, select the name of the column to use for the watermark. Values taken from this column are used to filter the contents of the next poll, so that rows only with a greater watermark value are processed.
```

Comment:
```suggestion
. On the *On Table Row* configuration screen, in *Watermark Column*, select the name of the column to use for the watermark. Values taken from this column are used to filter the contents of the next poll, so that only rows with a greater watermark value are processed.
```

Reply:
Interesting - usually a watermark is a visual (usually in the background of a window) that marks the identity of something. Here, it sounds like it is a number. Is this correct?